styles.css

```diff
@@ -0,0 +1,277 @@
+  background-color: var(--neutral-center);
+  border: none;
+  padding: 5px 25px;
+  margin: 5px;
+}
+
+.hire-me:hover {
+  opacity: 0.5;
+  color: var(--neutral-center);
+  background-color: var(--neutral);
+  border: 1px solid var(--neutral-center);
+}
+
+#about {
+  color: var(--base-100);
+  overflow: hidden;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 90%;
+  margin: 65px auto;
+}
+
+.her-content {
+  flex: 1;
+  width: 600px;
+  margin: 0 25px;
+}
+
+.her-image {
+  flex: 1;
+  width: 500px;
+  margin: auto;
+}
+
+img {
+  width: 100%;
+  height: auto;
+  border-radius: 10px;
+}
+
+.her-content h2 {
+  color: var(--neutral-center);
+  font-size: 28px;
+  padding-bottom: 10px;
+}
+
+.her-content p {
+  font-size: 24px;
+  line-height: 1.5;
+  margin-bottom: 40px;
+}
+
+.her-content button {
+  padding: 4px 20px;
+  color: var(--base-100);
+  background-color: var(--nature-green);
+}
+
+.her-content button:hover {
+  box-shadow: 5px 5px 10px hsl(0, 69%, 46%);
+  opacity: 0.7;
+}
+
+#projects {
+  background-color: var(--neutral-center);
+  color: var(--neutral);
+}
+
+#projects h1 {
+  padding-bottom: 20px;
+  text-align: center;
+  font-size: 34px;
+  border-bottom: 5px solid var(--neutral);
+  margin: auto;
+  width: fit-content;
+}
+
+#projects .project {
+  width: 100%;
+  height: 525px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 30px;
+  padding: 20px;
+  place-items: center;
+  max-width: 1200px;
+  margin: auto;
+  border-radius: 5px;
+}
+
+.project .project-tile {
+  width: 100%;
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: var(--base-100);
+  background-color: var(--neutral);
+  border: 5px;
+  box-shadow: 8px 8px 10px rgb(57, 25, 25);
+}
+
+#project .project-tile img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+
+#contact {
+  background-color: var(--neutral);
+  color: var(--base-100);
+  text-align: center;
+}
+
+#contact h1 {
+  font-size: 50px;
+  margin-top: 200px;
+}
+
+#contact .question {
+  color: var(--neutral-center);
+  font-size: 24px;
+  font-style: italic;
+}
+
+#contact .social-medias {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1000px;
+  justify-content: space-around;
+  margin: auto;
+  margin-top: 30px;
+}
+
+#contact .social-media {
+  text-decoration: none;
+  color: var(--base-100);
+  transition: transform 0.2s ease-in;
+  display: flex;
+  align-items: center;
+  margin: 30px;
+  font-size: 22px;
+}
+
+#contact .social-media:hover {
+  transform: translateY(10px);
+}
+
+footer {
+  border-top: 3px solid var(--neutral-center);
+  color: var(--base-100);
+  font-size: 25px;
+  padding: 30px;
+  text-align: center;
+  font-style: italic;
+}
+
+@media only screen and (width < 700px) {
+  h1 {
+    font-size: 50px;
+  }
+
+  #projects .project-tile {
+    height: 600px;
+    width: 100;
+  }
+}
```

Comment:
- You added only a few codes for your responsiveness, so your webpage does not look good in smaller versions.

![Screenshot 2024-06-04 115610](https://github.com/Hamas2000/Personal-Portfolio/assets/96838030/0b73dc84-53ed-4fd7-a661-25dc734b36ce)
![Screenshot 2024-06-04 115559](https://github.com/Hamas2000/Personal-Portfolio/assets/96838030/06a762cf-833c-4417-bcd9-0c4ec6c3844d)

![Screenshot 2024-06-04 115542](https://github.com/Hamas2000/Personal-Portfolio/assets/96838030/a949a803-de15-4672-a5b7-62c3890ec457)

Reply:
- Try to change the sizings and design of your project in different screen sizes.